pkgs/tools/misc/pcb2gcode/default.nix

```
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, boost
+, glibmm
+, gtkmm2
+, gerbv
+, librsvg
+, bash
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pcb2gcode";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "pcb2gcode";
+    repo = "pcb2gcode";
+    rev = "v${version}";
+    sha256 = "0nzglcyh6ban27cc73j4l7w7r9k38qivq0jz8iwnci02pfalw4ry";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkg-config bash ];
```

```suggestion
  nativeBuildInputs = [ autoreconfHook pkg-config ];
```

isn't bash part of the stdenv?

---

It is, I added it as an impromptu fix for the issues you were having with building it before. Not sure if it had any effect, but I can remove it.